Imagine/Filter/Loader/PasteFilterLoader.php

```diff
@@ -30,7 +30,7 @@ class PasteFilterLoader implements LoaderInterface
     public function __construct(ImagineInterface $imagine, $rootPath)
     {
         $this->imagine = $imagine;
-        $this->rootPath = $rootPath;
+        $this->rootPath = $rootPath . '/app';
```

Comment:
IMO this could be called $projectDir and do not append `/app` and then in the `load` function I would do something like:

```php
$file = $this->projectDir.'/'.$options['image'];

if (!file_exists($file) {
    @trigger_error(
        'The ' . $file . ' does not exists, change the path based on kernel.project_dir parameter',
        E_USER_DEPRECATED
    );
    $file = $this->rootPath.'/app/'.$options['image']
}
```